stable/dex/values.yaml

```diff
@@ -33,10 +33,17 @@ ports:
   - name: http
     containerPort: 8080
     protocol: TCP
+  - name: https
+    containerPort: 8443
+    port: 443
```

Comment:
Note that this does not validate against Deployment schema, as "port" does not exist in ContainerPort. It might be worth filtering this is deployment.yaml

Reply:
fixed